5calls/app/src/main/java/org/a5calls/android/a5calls/controller/RepCallActivity.java

```diff
@@ -153,7 +153,15 @@ public void onCallReported() {
         }
         setupContactUi(mActiveContactIndex, expandLocalOffices);
 
-        outcomeAdapter = new OutcomeAdapter(mIssue.outcomeModels, new OutcomeAdapter.Callback() {
+        // If the Issue's Outcome list is somehow empty, use default outcomes
```

Comment:
Optional: Point at a github bug where we track when this happens and anything we know about it.